tests/robot-tests/scripts/get_webdriver.py

```diff
@@ -1,23 +1,16 @@
 import os
-import platform
 from pathlib import Path
+from typing import Optional
 
-import pyderman
+from webdriver_manager.chrome import ChromeDriverManager
 
 
-def get_webdriver(version: str) -> None:
-    chromedriver_filename = "chromedriver.exe" if platform.system() == "Windows" else "chromedriver"
-    pyderman.install(
-        file_directory="./webdriver/",
-        filename=chromedriver_filename,
-        verbose=True,
-        chmod=True,
-        overwrite=False,
-        version=version,
-    )
+def get_webdriver(version: Optional[str] = None) -> None:
+    driver_path = ChromeDriverManager(driver_version=version).install()
```

Comment:
Where is chromedriver getting saved?

Having it downloaded to the webdrivers directory previously was useful to allow you to manually download a different version and use that rather than what the script downloads, which has happened on occasion.

Reply:
As mentioned in the PR summary, all webdrivers get downloaded to your home directory in `~/.wdm`. Each version downloaded gets its own directory, so you can just delete these as required.

However, I _think_ WDM by default will just try and figure out what Chrome you're using and get the correct version based on that. 

If you want to download a specific version, you should probably be using the `--chromedriver` flag instead when running the tests.